migrations/Version20200919002113.php

```diff
@@ -12,18 +12,18 @@
  */
 final class Version20200919002113 extends AbstractMigration
 {
-    public function getDescription() : string
+    public function getDescription(): string
```

Comment:
this should be run before commit to avoid having a dedicated commit at the end of the PR

Reply:
yes, I forgot it for this auto generated file...